src/Esfa.Vacancy.Register.Application/Queries/SearchApprenticeshipVacancies/SearchApprenticeshipVacanciesRequestValidator.cs

```diff
@@ -0,0 +1,24 @@
+﻿using FluentValidation;
+
+namespace Esfa.Vacancy.Register.Application.Queries.SearchApprenticeshipVacancies
+{
+    public class SearchApprenticeshipVacanciesRequestValidator : AbstractValidator<SearchApprenticeshipVacanciesRequest>
+    {
+        public SearchApprenticeshipVacanciesRequestValidator()
+        {
+            RuleFor(r => r.StandardCodes)
+                .NotNull()
+                .WithMessage("At least one search parameter is required.");
+
+            RuleForEach(r => r.StandardCodes)
+                .Must(BeValidNumber)
```

Comment:
This is good. I am just thinking should we check it is a positive number. This is only because I know of the horrors around some of the Id's in the database on tables going into negative values. I don't think it is the case for Standards but thought I would mention it. Not intending to overcomplicate.